x-pack/plugins/ml/public/embeddables/lens/lens_vis_layer_selection_flyout/style.scss

```diff
@@ -0,0 +1,23 @@
+// rough duplicates of lens styles
+// so our layer list looks similar
+
+.mlLensToJobFlyoutBody {
+  background-color: lightOrDarkTheme($euiColorLightestShade, $euiColorInk);
+}
+
+.mlLnsLayerPanel__layerHeader {
+  padding: $euiSize;
+  border-bottom: $euiBorderThin;
+}
+
+.mlLnsLayerPanel__settingsStaticHeader {
+  padding-left: $euiSizeXS;
+}
+
+.mlLnsLayerPanel__row {
+  padding: $euiSize;
+}
+
+.mlSubdued {
```

Comment:
This would be removed if you switch to using the `EuiText` component and use the `color` prop.

Reply:
Changed in https://github.com/elastic/kibana/pull/129762/commits/82accec37a8e4b8790722f38f87fed1fae3deb55